### ISSUE TITLE
Brew install errors out during install

### ISSUE THREAD

**Issue by isoung** [amberframework/homebrew-amber#1]
Steps to Reproduce --
1. brew tap amberframework/homebrew-amber
2. brew install amber

>==> make install PREFIX=/usr/local/Cellar/amber/0.8.0
>Last 15 lines from /Users/isaiahsoung/Library/Logs/Homebrew/amber/02.make:
>require "./commands/*"
>^
>in src/amber/cli/commands/pipelines.cr:2: while requiring "shell-table"
>require "shell-table"
>^
>Syntax error in lib/shell-table/src/shell-table.cr:30: setter method 'labels=' cannot have more than >one argument
>  def labels=(*args, **props)
>                            ^
>make: *** [/private/tmp/amber-20180821-60911-1ayid83/amber-0.8.0/bin/amber] Error 1
>If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
>https://github.com/amberframework/homebrew-amber/issues

**Comment by rmgas1** [comment]
Hi @isoung, this seems to be resolved for v0.10.0:
```
$ brew tap amberframework/amber
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
No changes to formulae.

==> Tapping amberframework/amber
Cloning into '/usr/local/Homebrew/Library/Taps/amberframework/homebrew-amber'...
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 5 (delta 0), reused 3 (delta 0), pack-reused 0
Unpacking objects: 100% (5/5), done.
Tapped 1 formula (30 files, 27.2KB).

$ brew install amber
==> Installing amber from amberframework/amber
==> Downloading https://github.com/amberframework/amber/archive/v0.10.0.tar.gz
==> Downloading from https://codeload.github.com/amberframework/amber/tar.gz/v0.10.0
######################################################################## 100.0%
==> shards install
==> make install PREFIX=/usr/local/Cellar/amber/0.10.0
🍺  /usr/local/Cellar/amber/0.10.0: 5 files, 6.3MB, built in 1 minute 20 seconds
```

**Comment by robacarp** [comment]
thanks @rmgas1
